src/modm/container/doubly_linked_list_impl.hpp

```diff
@@ -33,8 +33,8 @@ modm::DoublyLinkedList<T, Allocator>::~DoublyLinkedList()
 		Node *node = this->front;
 		this->front = this->front->next;
 
-		Allocator::destroy(&node->value);
-		this->nodeAllocator.deallocate(node);
+		std::destroy_at(&node->value);
+		this->nodeAllocator.deallocate(node, 1);
```

Comment:
This should use [`std::allocator_traits<Allocator>::destroy`](https://en.cppreference.com/w/cpp/memory/allocator_traits/destroy). It falls back to `std::destroy_at` in case the allocator does not customize `destroy`.